test/dynamo/test_functions.py

```diff
@@ -786,7 +786,7 @@ def fn(x):
         self.assertTrue(same(ref[1][param], res[1][param]))
 
     def test_dict_tuple_lazy_guard(self):
-        @torch.compile(backend="eager")
+        @torch.compile(backend=self.backend)
```

Comment:
A lot of these tests might not work under a different backend. For example, if a test counts the number of guards, then using eager vs aot_eager would likely lead to more.

Reply:
Yes - I'd like those to fail and then get excluded. I plan to stack enablements on this 1 file at a time.